### PR TITLE
Landing page contributions

### LINKED ISSUE #9: Adding a landing page
User Story:
As a “Shlifim” website user, I want to search or upload questions I may have from homework or assignments I have from school so that I can receive answers, summaries, or sources for my questions.
I'll know I'm done when I managed to find my answer and finish my homework or assignments.

Use Case:
A new user will sign up to the website while a registered user will log in.
Once the user is on the 'Home Page' he will be able to look through the latest questions.
He will be able to search and filter existing questions, add a new question, or add a new answer to an existing question.

For Demo 1 we will create a landing page that includes:
- Our application logo
- A brief explanation of the app
- Coming soon features
- Team members

### PR DESCRIPTION
**Pull request by avivz450**
- Added a landing page to our website - project introduction page
- Added landingpage directory
- Edited shlifim/settings.py shlifim/urls.py

fix #9

### CODE REVIEW

**Comment by ifireball** [comment]
Why do you call this "demo page"? It makes it sound like you would remove this entire thing in the near future - wouldn't it be better to call it the "landing page" because its essentially the first page of the application?

**Comment by avivz450** [comment]
> essentially

**Comment by avivz450** [comment]
Closed by a mistake... I sounds more

File: demopage/templates/demopage/app.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+body {
+    background-color: rgb(218, 214, 214);
+    font-family: Montserrat, sans-serif;
+}
```
[the remark this answers is not in the document]
Comment (avivz450) [reply demopage/templates/demopage/app.css]: Thanks for the update, I'll change that

**Comment by liornoy** [comment]
LGTM

File: demopage/templates/demopage/app.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (avivz450) [reply demopage/templates/demopage/app.css]: Done

**Comment by AviadP** [comment]
As general practice, please link PR to relevant issue

**Comment by ifireball** [comment]
I see a mix of autogeneraed code and manually written code in this PR - please make sure those go in separate commits or even separate PRs.

**Comment by danitLevi** [comment]
I think this PR can be merged

**Comment by ifireball** [comment]
Im still seeing `landingpage/templates/landingpage/landingpage.html` which is manually written, mixed in the same commit with auto-generated files like `app.py` and `admin.py` and `models.py` etc.

**Comment by avivz450** [comment]
> Im still seeing `landingpage/templates/landingpage/landingpage.html` which is manually written, mixed in the same commit with auto-generated files like `app.py` and `admin.py` and `models.py` etc.

`app.py`, `admin.py` and `models.py` were auto-generate, but then I edited them manually.
Do they still need to be on the autogenerated code commit?

**Comment by ifireball** [comment]
The fact that you edited manually is more reason why you have committed the generated code before starting to edit anything - Otherwise its very hard for me to know what the manual edits are unless I remember how the generated code looks "by heart".

**Comment by avivz450** [comment]
Ok, I will make a new PR

**Pull request closed without merging**